File: 0066-plus-one/0066-plus-one.js
/**
 * @param {number[]} digits
 * @return {number[]}
 */
var plusOne = function(digits) {
    let num =BigInt(digits.join(""))
    num=num+1n
    let res=[]
    while(num>0n){
        res.push(Number(num%10n))
        num = num/10n
    }
    return (res.reverse())
};

File: 0066-plus-one/0066-plus-one.go
func plusOne(digits []int) []int {
    addCarry(&digits, len(digits)-1)
    return digits
}

func addCarry(digits *[]int, i int) {
    if i < 0 {
        *digits = append([]int{1}, *digits...)
        return
    }

    if (*digits)[i] < 9 {
        (*digits)[i]++
        return
    }

    (*digits)[i] = 0
    addCarry(digits, i-1)
}
